app/grandchallenge/api/urls.py

```diff
@@ -74,7 +86,7 @@
         description=f"The API for {settings.SESSION_COOKIE_DOMAIN.lstrip('.')}.",
         license=openapi.License(name="Apache License 2.0"),
     ),
-    permission_classes=(permissions.IsAuthenticated,),
+    permission_classes=(IsSuperUserOrReadOnly,),
```

Comment:
I'm not sure why we would ever want to POST or PATCH to this view. All of this can probably be replaced with
```suggestion
    permission_classes=(permissions.AllowAny,),
    public=True,
```

See https://drf-yasg.readthedocs.io/en/stable/readme.html#quickstart

Reply:
That will also work, I just wanted to enforce read-only permissions for all users for safety if it ever changes in the future.

Reply:
I think it would be fairly safe, the API for this method is clear and follows a good deprecation cycle, there's no DB table for this app to even edit here.

Reply:
Permissions on the view are now AllowAny